ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/po/cucushift-installer-rehearse-aws-ipi-proxy-po-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
```

Comment:
@JoelSpeed 
add @jianzhangbjz review it because the original workflow is done by him.